charts/warm-metal-csi-driver/templates/nodeplugin.yaml

```diff
@@ -122,7 +122,7 @@ spec:
             - mountPath: {{ .Values.crioMountProgram }}
               name: crio-mount-program
             {{- end }}
-      hostNetwork: true
+      hostNetwork: {{.Values.hostNetwork}}
```

Comment:
Shouldn't the variable be `.Values.csiPlugin.hostNetwork`?

Reply:
Good catch. Fixed.